src/initializations.rs

```diff
@@ -53,6 +53,13 @@ pub fn uniform<T: HasAfEnum>(dims: Dim4, min: f32, max: f32) -> Array{
   u
 }
 
+// Provide values between 0 and "scale" from a uniform distribution 
+pub fn uniform_pos<T: HasAfEnum>(dims: Dim4, scale: f32) -> Array{
```

Comment:
FYI: I have modified the `uniform` call here so that you can pass in a range `[a, b]`